consensus/broadcast/delayedBroadcast_test.go

```diff
@@ -299,6 +315,83 @@ func TestDelayedBlockBroadcaster_HeaderReceivedForNotRegisteredDelayedDataShould
 	assert.False(t, txBroadcastCalled.IsSet())
 }
 
+func TestDelayedBlockBroadcaster_HeaderReceivedWithoutSignaturesForShardShouldNotBroadcastTheData(t *testing.T) {
+	var logOutput bytes.Buffer
+	var logMutex sync.Mutex
+
+	customFormatter := &logger.PlainFormatter{}
+	observer := syncLogObserver{
+		buffer:    &logOutput,
+		mutex:     &logMutex,
+		formatter: customFormatter,
+	}
+	err := logger.AddLogObserver(&observer, customFormatter)
+	require.Nil(t, err)
```

Comment:
you can create a new method that initializes the observer, thus these lines will become:
```go
observer := createLogsObserver()
err := logger.AddLogObserver(observer, &logger.PlainFormatter{})
require.Nil(t, err)
```

func example:
```go
func createLogsObserver() *syncLogObserver {
	return &syncLogObserver{
		buffer: &bytes.Buffer{},
		mutex:  &sync.Mutex{},
	}
}
```

Reply:
this can be reused in all tests